cognee/cli/_cognee.py

```
@@ -0,0 +1,180 @@
+import sys
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)